bamboo_id.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 1.14"
+  spec.add_development_dependency 'bundler', '~> 2.2.18'
```

Comment:
Why the change in quotes?

Reply:
This was copied from bonusly/tangolicious#1